src/ActionsImporter/App.cs

```diff
@@ -12,6 +12,12 @@ public class App
     private readonly IProcessService _processService;
     private readonly IConfigurationService _configurationService;
 
+    public bool IsPrerelease { get; set; }
+
+    private string ImageTag => IsPrerelease ? "pre" : "latest";
+
+    private string ImageName => $"{ActionsImporterImage}:{ImageTag}";
```

Comment:
It looks like this has a downstream effect on the `version` subcommand. 👇 compares the output vs the output on `main`

```shell
gh-actions-importer on  main
$ dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- version
A new version of GitHub Actions Importer is available. Run 'gh actions-importer update' to update.
gh version 2.18.1 (2022-10-20)
gh actions-importer	github/gh-actions-importer	v1.0.2
actions-importer/cli	v1.0.14888

$ gh pr checkout 39
Switched to branch 'support-prerelease-flag'

gh-actions-importer on  support-prerelease-flag
$ dotnet run --project src/ActionsImporter/ActionsImporter.csproj -- version
A new version of GitHub Actions Importer is available. Run 'gh actions-importer update' to update.
gh version 2.18.1 (2022-10-20)
gh actions-importer	github/gh-actions-importer	v1.0.2
actions-importer/cli:latest	unknown
```

Reply:
Ah! This should be fixed now. I accidentally removed the repository from the version handler in `App.cs`. That `RunAndCaptureAsync` doesn't throw on error, so it didn't bubble up.

Should be fixed now! I also double checked all the other calls to Docker to make sure they pass the registry, but all of the other commands do throw errors, so they'd be easier to catch.